packages/flutter_hooks/lib/src/framework.dart

```diff
@@ -353,6 +372,7 @@ mixin HookElement on ComponentElement {
 
   _Entry<HookState>? _currentHookState;
   final _hooks = LinkedList<_Entry<HookState>>();
+  final _needDidBuildHooks = <HookState>{};
```

Comment:
Could this be refactoring to a `List`, and have this property nullable by default? for performance

We wouldn't want to instantiate a list for every hook widget when few will actually care about this.

Reply:
Why a ´List` ? Why can't we keep the `Set` but make it nullable ?

Reply:
This is done but I've kept the `Set`, let me know if you really want a list here.